src/com/google/caja/plugin/caja.js

```diff
@@ -23,6 +23,29 @@
  */
 
 var caja = (function () {
+  // As of January 31st, 2021, the Caja project is going to be deprecated 
+  // Print a note about it in the browser's console
+  var deprecation_message = [
+    'On January 31st, 2021, we will be archiving the Caja project. ',
+    'After January 31, no new features will be added, pull requests and other ',
+    'issues will no longer be addressed, including patches for security issues, ',
+    'and the repository will be marked as archived. Caja has not been actively ',
+    'maintained or developed to keep up with the latest research on web security. ',
+    'As a result, several security vulnerabilities have been reported to Caja, ',
+    'both by Google’s security engineers and by external researchers.\n\n',
+    'We encourage Caja users to migrate to Closure toolkit ', 
```

Comment:
Anyone who is loading `caja.js` (as opposed to `html-sanitizer.js`) is quite likely intending to support embedded JavaScript. "Migrate to Closure's sanitizers" is not a replacement for those use cases. Please rephrase this to at least *acknowledge* that this recommendation is only relevant to non-script-containing content, whether or not there is any recommendation for a replacement.

Reply:
P.S.: this _would_ be an appropriate message to embed in `html-sanitizer.js` and `html-sanitizer-legacy.js`.